run-tests-with-docker.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+#######################################
+# The script starts remote-chrome or remote-firefox docker container for running Browser tests on a developer machine only.
+# The container directs TCP requests from the container to the host machine enabling testing services running via Service Manager.
+# WARNING: Do not use this script in the Jenkins Continuous Integration environment
+#
+# Arguments:
+#   remote-chrome or remote-firefox
+#
+# Output:
+#   Starts  chrome or firefox docker containers from chrome-with-rinetd or firefox-with-rinetd image respectively
+#######################################
+
+#######################################
+# Requires services under test running via Service Manager
+# Initializes port_mappings with all running application ports using the Service Manager status command.
+# Appends ZAP_PORT 11000 to ./run-zap-spec.sh
+#######################################
+port_mappings=$(sm -s | grep PASS | awk '{ print $12"->"$12 }' | paste -sd "," -)
+port_mappings="$port_mappings,11000->11000,9557->9557,9553->9553,9554->9554,9556->9556,9597->9597"
+
+# Alternatively, port_mappings can be explicitly initialised as below:
+#port_mappings="9032->9032,9250->9250,9080->9080"
+
+#######################################
+# Defines the BROWSER variable from the argument passed to the script
+#######################################
+if [ -z "${1}" ]; then
+  echo "ERROR: Browser type not specified. Re-run the script with the option remote-chrome or remote-firefox."
+  exit 1
+elif [ "${1}" = "remote-chrome" ]; then
+  BROWSER="artefacts.tax.service.gov.uk/chrome-with-rinetd:94-0.7.0"
+elif [ "${1}" = "remote-firefox" ]; then
+  BROWSER="artefacts.tax.service.gov.uk/firefox-with-rinetd:92-0.6.0"
```

Comment:
Just wondering if instead of specifying a specific version or browser driver there is a possibility to use the latest one.

Locally the chrome version sometimes is updated automatically. Additionally, probably we should always run the tests against the latest chrome/firefox driver, so with this approach, we will need to update this file periodically.

Reply:
There is a list of images available https://artefacts.tax.service.gov.uk/ui/repos/tree/General/chrome-with-rinetd%2Fchrome-with-rinetd I just picked the latest version for firefox and chrome. Not sure if there is a way to specify "always latest".